src/test/java/io/jenkins/plugins/pipelinegraphview/utils/TestUtils.java

```diff
@@ -25,24 +25,38 @@ public class TestUtils {
     private static final Logger LOGGER = Logger.getLogger(TestUtils.class.getName());
 
     public static WorkflowRun createAndRunJob(
-            JenkinsRule jenkins, String jobName, String jenkinsFileName, Result expectedResult) throws Exception {
-        WorkflowJob job = TestUtils.createJob(jenkins, jobName, jenkinsFileName);
+        JenkinsRule jenkins, String jobName, String jenkinsFileName, Result expectedResult) throws Exception {
+            return createAndRunJob(jenkins, jobName, jenkinsFileName, expectedResult, true);
+    }
+
+    public static WorkflowRun createAndRunJob(
+            JenkinsRule jenkins, String jobName, String jenkinsFileName, Result expectedResult, boolean sandbox) throws Exception {
+        WorkflowJob job = TestUtils.createJob(jenkins, jobName, jenkinsFileName, sandbox);
         jenkins.assertBuildStatus(expectedResult, job.scheduleBuild2(0));
         return job.getLastBuild();
     }
 
     public static QueueTaskFuture<WorkflowRun> createAndRunJobNoWait(
             JenkinsRule jenkins, String jobName, String jenkinsFileName) throws Exception {
-        WorkflowJob job = TestUtils.createJob(jenkins, jobName, jenkinsFileName);
+        return createAndRunJobNoWait(jenkins, jobName, jenkinsFileName, true);
+    }
+
+    public static QueueTaskFuture<WorkflowRun> createAndRunJobNoWait(
+            JenkinsRule jenkins, String jobName, String jenkinsFileName, boolean sandbox) throws Exception {
+        WorkflowJob job = TestUtils.createJob(jenkins, jobName, jenkinsFileName, sandbox);
         return job.scheduleBuild2(0);
     }
 
     public static WorkflowJob createJob(JenkinsRule jenkins, String jobName, String jenkinsFileName) throws Exception {
+        return createJob(jenkins, jobName, jenkinsFileName, true);
+    }
+
+    public static WorkflowJob createJob(JenkinsRule jenkins, String jobName, String jenkinsFileName, boolean sandbox) throws Exception {
         WorkflowJob job = jenkins.createProject(WorkflowJob.class, jobName);
 
         URL resource = Resources.getResource(TestUtils.class, jenkinsFileName);
         String jenkinsFile = Resources.toString(resource, Charsets.UTF_8);
-        job.setDefinition(new CpsFlowDefinition(jenkinsFile, true));
+        job.setDefinition(new CpsFlowDefinition(jenkinsFile, sandbox));
```

Comment:
Expose the sandbox property as a parameter as the test `gh616_stageResult.jenkinsfile` uses [`FlowNode#getEnclosingBlocks()`](https://github.com/jenkinsci/workflow-api-plugin/blob/e334280bd61171465c783d5c34efe9e7f634a6af/src/main/java/org/jenkinsci/plugins/workflow/graph/FlowNode.java#L194-L196) to look up parent stages.